EIPS/eip-2535.md

```diff
@@ -0,0 +1,465 @@
+---
+eip: 2535
+title: Diamond Standard
+author: Nick Mudge <nick@perfectabstractions.com>
+discussions-to: https://github.com/ethereum/EIPs/issues/2535
+status: Draft
+type: Standards Track
+category: ERC
+created: 2020-02-22
+---
+
+<img align="right" src="../assets/eip-2535/diamond.svg" width="230" height="230">
+
+## Simple Summary
+
+A diamond is a set of contracts that can access the same storage variables and share the same Ethereum address.
+
+A contract architecture that makes upgradeable contracts flexible, unlimited in size, and transparent. 
+
+New terminology from the diamond industry.
+
+Improved design over [ERC1538](https://eips.ethereum.org/EIPS/eip-1538) using ABIEncoderV2 and function selectors.
+
+Replaces ERC1538.
```

Comment:
Imagine you are reading this standard 10 years in the future.  These lines would not add value or improve your understanding of the standard, so IMO they should not be included.

Reply:
That is a good point and you are right.  I'll move the mentions of ERC1538 to the bottom of the EIP under inspiration & development.